example/extract/extract_html.ipynb

```diff
@@ -0,0 +1,235 @@
+   "outputs": [],
+   "source": [
+    "data = [{\"filename\": f'../transform/data/raw_input/22.11_information-theory.html'}]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the html file via ExtractClient"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = ExtractClient(ExtractHTMLConfig())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "  0%|          | 0/1 [00:00<?, ?it/s]"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 1/1 [00:00<00:00,  4.53it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "output = client.run(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Output\n",
+    "\n",
+    "Let's take a look of the generation output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "['22.11. Information Theory — Dive into Deep Learning 1.0.3 documentation22.',\n",
+      " 'Appendix: Mathematics for Deep Learning',\n",
+      " '2.1. Data Manipulation',\n",
+      " '2.2. Data Preprocessing',\n",
+      " '2.5. Automatic Differentiation',\n",
+      " '2.6. Probability and Statistics',\n",
+      " '3. Linear Neural Networks for Regression',\n",
+      " '3.1. Linear Regression',\n",
+      " '3.2. Object-Oriented Design for Implementation',\n",
+      " '3.3. Synthetic Regression Data',\n",
+      " '3.4. Linear Regression Implementation from Scratch',\n",
+      " '3.5. Concise Implementation of Linear Regression',\n",
+      " '4. Linear Neural Networks for Classification',\n",
+      " '4.1. Softmax Regression',\n",
+      " '4.2. The Image Classification Dataset',\n",
+      " '4.3. The Base Classification Model',\n",
+      " '4.4. Softmax Regression Implementation from Scratch',\n",
+      " '4.5. Concise Implementation of Softmax Regression',\n",
+      " '4.6. Generalization in Classification',\n",
+      " '4.7. Environment and Distribution Shift']\n"
+     ]
+    }
+   ],
+   "source": [
+    "text = output[0]['output'][0]['text'][0]\n",
+    "text = [p for p in text.split(\"\\n\") if len(p) > 20]\n",
+    "pprint.pprint(text[:20])"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "uniflow",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
nit: follow https://github.com/CambioML/uniflow/blob/main/example/extract/extract_pdf.ipynb to add a footer for this notebook.